testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -1101,6 +1101,31 @@ def test_missing_icodes_range_VE(self, u):
             u.select_atoms('resid 10A-12')
 
 
+@pytest.mark.parametrize(
+    "selection, n_atoms", 
+    [
+        # Selection using resindices
+        # For PDBs: residues with different insertion codes have different resindices
+        ("same residue as ", 11),
+        # Selection using resids
+        # Residues with different insertion codes have the same resid
+        # See Issues #2308 for a discussion
+        ("same resid as", 72),
+        # Selection using resindices
+        # For PDBs: residues with different insertion codes have different resindices
+        ("byres", 11)
+    ]
+)
+def test_similarity_selection_icodes(selection, n_atoms):
+
+    u = mda.Universe(PDB_icodes)
```

Comment:
make this a fixture

Reply:
@orbeckst Done (if what I did is what you had in mind).